std_msgs/Int32[] previousFrameIndexes
std_msgs/Int32[] currentFrameIndexes
std_msgs/String previousFrame
std_msgs/String currentFrame
